raphtory-graphql/src/data.rs

```diff
@@ -45,31 +42,29 @@ impl Data {
         let graphs_cache: Arc<Cache<String, IndexedGraph<MaterializedGraph>>> =
             Arc::new(graphs_cache_builder);
 
-        save_graphs_to_work_dir(work_dir, &graphs.unwrap_or_default())
-            .expect("Failed to save graphs to work dir");
-
-        load_graphs_from_paths(work_dir, graph_paths.unwrap_or_default(), true)
-            .expect("Failed to save graph paths to work dir");
-
         Self {
             work_dir: work_dir.to_string_lossy().into_owned(),
             graphs: graphs_cache,
             global_plugins: GlobalPlugins::default(),
         }
     }
 
-    pub fn get_graph(&self, name: &str) -> Result<Option<IndexedGraph<MaterializedGraph>>> {
+    pub fn get_graph(
+        &self,
+        name: &str,
+        namespace: &Option<String>,
```

Comment:
this should be `Option<&str>` I think. Making an `&Option<String>` is hard because you already need to have an `Option<String>` to reference. You can always make `Option<&str>` as long as you have any kind of string-like thing (including `Option<String>`)